relayer/LICENSE

```diff
@@ -0,0 +1,165 @@
+                   GNU LESSER GENERAL PUBLIC LICENSE
```

Comment:
I think this may have been due to originally using some ChainBridge files for our key management, but we don't really use any code from them anymore. we could change this to apache 2

Reply:
Okay, I'll leave this as is for now, please create an issue if you think it's worthwhile to change?